parser/v2/types.go

```diff
@@ -375,17 +375,27 @@ func (t HTMLTemplate) Write(w io.Writer, indent int) error {
 type TrailingSpace string
 
 const (
-	SpaceNone       TrailingSpace = ""
-	SpaceHorizontal TrailingSpace = " "
-	SpaceVertical   TrailingSpace = "\n"
+	SpaceNone           TrailingSpace = ""
+	SpaceHorizontal     TrailingSpace = " "
+	SpaceVertical       TrailingSpace = "\n"
+	SpaceVerticalDouble TrailingSpace = "\n\n"
 )
 
 var ErrNonSpaceCharacter = errors.New("non space character found")
 
-func NewTrailingSpace(s string) (ts TrailingSpace, err error) {
+func NewTrailingSpace(s string, allowMulti bool) (ts TrailingSpace, err error) {
 	var hasHorizontalSpace bool
-	for _, r := range s {
+
+	runes := []rune(s)
+
+	for i, r := range s {
 		if r == '\n' {
+			if allowMulti && i < len(runes)-1 {
+				next := runes[i+1]
+				if next == '\n' {
```

Comment:
I had assumed this was normalized before entering this part of the code. Seems dangerous to have 2 code-paths based on OS everywhere in the code. I'd check, but it'd be difficult for me to test it since I don't have Windows :sweat_smile: